examples/ResNet/ResNet.cpp

```diff
@@ -332,9 +333,16 @@ const wnn::Operand ResNet::LoadNCHW(const wnn::GraphBuilder& builder, bool softm
     return output;
 }
 
-const wnn::Operand ResNet::LoadNHWC(const wnn::GraphBuilder& builder, bool softmax) {
-    mWeightsPath = mWeightsPath + "resnet_v2_101_";
-    const wnn::Operand input = utils::BuildInput(builder, "input", {1, 299, 299, 3});
+// Create ResNet50V2/ResNet101V2 nhwc model.
+const wnn::Operand ResNet::LoadNhwc(const wnn::GraphBuilder& builder, bool softmax, bool is50V2) {
+    wnn::Operand input;
+    if (is50V2) {
+        mWeightsPath = mWeightsPath + "resnet_v2_50_";
+        input = utils::BuildInput(builder, "input", {1, 224, 224, 3});
+    } else {
+        mWeightsPath = mWeightsPath + "resnet_v2_101_";
```

Comment:
Remove the `bool is50V2101`  parameter because 101 is not our foucs, you can set the mWeightsPath / mModelHeight / mModelWith in https://github.com/webmachinelearning/webnn-native/pull/251/files#diff-b0d16df9750ad386eeba6333ee9d68a4066542a8856ce33abc9a2a202a40429bR28